compute_tools/src/spec.rs

```diff
@@ -684,6 +684,14 @@ pub fn handle_grants(
         );
         db_client.simple_query(&grant_query)?;
 
+        for entity in ["TABLES", "SEQUENCES"] {
+            let grant_query = format!(
+                "ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT ALL ON {} TO neon_superuser",
+                entity
+            );
+            db_client.simple_query(&grant_query)?;
+        }
+
```

Comment:
If we do decide to go this route of default privileges, I think it would be better to move this to `handle_migrations` instead

Reply:
fixed in https://github.com/neondatabase/neon/pull/6845/commits/36052ada318d0b19965f036b308dde5ea3e79a04

Reply:
Thanks! Don't forget to also bump the `num_migrations` variable in `test_migrations`.